### PR TITLE
added a pop up box to compose text

### PR DESCRIPTION
**Pull request by frankmfernandez**
added a pop up box to compose text

will note that there is a subject in the form now which is not connected to anything, happy to remove. the form currently updates messages similar to how it did previously by updating the name column.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -31,6 +31,8 @@
     <title>React App</title>
   </head>
   <body>
+    <div id="fb-root"></div>
+    <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v4.0"></script>
```
Comment (wilandrade) [inline public/index.html:35]: Why is this needed for the text popup?

File: src/components/TextUpload1.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+class TextUpload1 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      upload: ""
+    };
+  }
+  closeForm(){
+    console.log("the form is closing")
```
Comment (wilandrade) [inline src/components/TextUpload1.jsx]: Please remove console.logs

File: src/components/TextUpload1.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      campaignId: 1 // Set to default 1st campaign
+    };
+    console.log("this is state upload", this.state.upload)
+    this.props.createNewMedia(media);
+    this.closeForm()
+  }
+
+  render() {
+    return (
+      // <a className="nav-link" href="#home">
```
Comment (wilandrade) [inline src/components/TextUpload1.jsx]: Pleaes remove comments

File: src/index.css [hunk 4]
```diff
@@ -21,3 +21,46 @@ code {
...
+  position:fixed;
+  left:0;
+  top:0;
+  width:100vw;
+  height:100vh;
+  display:none;
+  background-color:rgba(14, 47, 90, 0.6);
+  z-index: 6;
+
+ /* // opacity:0.5; */
```
Comment (wilandrade) [inline src/index.css]: please remove comment

File: src/components/Share.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    super(props);
+    this.state = {
+      
+    };
+   
+  }
+
+  render() {
+    return (
+      <div class="fb-share-button" data-href="https://espn.com" data-layout="button" data-size="large">
```
Comment (wilandrade) [inline src/components/Share.jsx:15]: I'm not sure this feature is useful for us. I'm happy you challenges yourself technically but this would only be useful, in my opinion, if we were able to get deeplinks on the app side to work.As it stands, I think we don't have enough time to get that done.

**Review by wilandrade — CHANGES_REQUESTED** [review]
Please see change requests. And maybe let's talk as a team whether or not we need the share stuff. I think it might be unnecessary.

File: public/index.html (hunk 1 above)
Comment (frankmfernandez) [reply public/index.html:35]: this is for the fb share button.

**Comment by frankmfernandez** [comment]
> Please see change requests. And maybe let's talk as a team whether or not we need the share stuff. I think it might be unnecessary.

totally fine to remove it, just took a stab at it this morning, we can discuss

File: src/components/TextUpload1.jsx (hunk 2 above)
Comment (frankmfernandez) [reply src/components/TextUpload1.jsx]: ok

File: src/components/TextUpload1.jsx (hunk 3 above)
Comment (frankmfernandez) [reply src/components/TextUpload1.jsx]: ok

File: src/index.css (hunk 4 above)
Comment (frankmfernandez) [reply src/index.css]: ok

File: src/components/Share.jsx (hunk 5 above)
Comment (frankmfernandez) [reply src/components/Share.jsx:15]: cool and totally fine with me.

File: src/components/TextUpload1.jsx (hunk 2 above)
Comment (wilandrade) [reply src/components/TextUpload1.jsx]: I see. Cool

File: public/index.html (hunk 1 above)
Comment (wilandrade) [reply public/index.html:35]: I see, cool

File: src/components/Share.jsx (hunk 5 above)
Comment (wilandrade) [reply src/components/Share.jsx:15]: We'll keep this for now as a way for people ( and us) to share the portal with other people.

File: src/index.css [hunk 6]
```diff
@@ -21,3 +21,47 @@ code {
...
+}
+
+.h4{
+  color: #0E2F5A;
+  margin-top:4%;
+}
+
+#fb-share-button{
+  margin:0 auto;
+}
```
Comment (wilandrade) [inline src/index.css]: just need to add a newline at the end of the file here and we're golden.

**Review by wilandrade — CHANGES_REQUESTED** [review]
